<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiosSearch</class>
 <widget class="QWidget" name="BiosSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/search.svg</normaloff>:/search.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="textTab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <widget class="QLabel" name="SearchTitle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>7</y>
         <width>361</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search FV/FFS/Section Name:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>154</width>
         <height>74</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="FvCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Search FV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="FfsCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Search FFS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="SectionCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Search Section</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>112</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="CaseCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Case Sensitive</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QComboBox" name="SearchContentBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="guidTab">
      <attribute name="title">
       <string>GUID</string>
      </attribute>
      <widget class="GuidData1HexLineEdit" name="Data1Edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Data2Edit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>40</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Data3Edit">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>40</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Data4Edit">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>40</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Data5Edit">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>100</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>329</x>
         <y>10</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QLabel" name="SearchTitle_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search GUID in FV/FFS/Section:</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>154</width>
         <height>74</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="GuidFvCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Search FV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="GuidFfsCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Search FFS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="GuidSectionCheckbox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Search Section</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>Data1Edit</zorder>
      <zorder>Data2Edit</zorder>
      <zorder>Data3Edit</zorder>
      <zorder>Data5Edit</zorder>
      <zorder>clearButton</zorder>
      <zorder>SearchTitle_2</zorder>
      <zorder>layoutWidget_2</zorder>
      <zorder>Data4Edit</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PreviousButton">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuidData1HexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Input/inputdialog.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../logo/Images.qrc"/>
 </resources>
 <connections/>
</ui>
